ping_server.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+ 
+# -q quiet
+# -c number of pings to perform
+ 
+ping -q -c2 google.com > /dev/null
```

Comment:
Not sure why this pings `google.com`. It should ping our server by simulating a request.

```
curl https://opinion.berkeley.edu/pcari/en/landing/
```

Reply:
WHOOPS I meant to change to opinion